EvotoApi/Areas/Management/Controllers/RegistrarConnectionController.cs

```diff
@@ -128,5 +128,42 @@ public class RegistrarConnectionController : ApiController
                 });
             }
         }
+
+        [Route("settings/list")]
+        //[Authorize]
+        [HttpGet]
+        public async Task<IHttpActionResult> ListSettings()
+        {
+            try
+            {
+                var settings = await RegistrarConnection.ListRegistrarSettings();
+                return Json(settings);
```

Comment:
Ok() is dynamic and responds to the contentType of the request. Avoids unnecessary dependence on json